frontend/package.json

```diff
@@ -26,7 +26,7 @@
         "@heroicons/react": "^1.0.6",
         "@monaco-editor/react": "^4.4",
         "@primer/octicons-react": "^17",
-        "@redpanda-data/ui": "^2.24.0",
+        "@redpanda-data/ui": "^2.40.0",
```

Comment:
There's a small issue with the sidebar in this version, if you approve my PR to revert it, we can use 2.41